ddpui/api/user_org_api.py

```
@@ -624,3 +628,225 @@ def get_organization_wren(request):
+        httponly=True,
+        secure=secure_cookies,
+        samesite="lax",
+        path="/",
+    )
+
+    return response
+
+
+@user_org_router.get("/v2/currentuser", response=List[OrgUserResponse])
+@has_permission(["can_view_orgusers"])
+def get_current_user_v2(request, org_slug: str = None):
```

Ruff (0.13.3)

Redefinition of unused `get_current_user_v2` from line 65

(F811)